cli/version/version.go

```diff
@@ -17,15 +21,75 @@ var (
 )
 
 const (
-	unknownVersion = "<unknown>"
-	cliName        = "Tarantool Cartridge CLI"
+	unknownVersion        = "<unknown>"
+	cliName               = "Tarantool Cartridge CLI"
+	cartridgeVersionTitle = "Tarantool Cartridge"
+	errorStr              = "Failed to get the version of the Cartridge"
 )
 
-func BuildVersionString() string {
-	var version string
+func format(template string, templateArgs map[string]string) string {
+	versionMsg, err := templates.GetTemplatedStr(&template, templateArgs)
+
+	if err != nil {
+		panic(err)
+	}
+
+	return versionMsg
+}
+
+func getRocksVersions(projectPath string) (map[string]string, error) {
+	var rocksVersionsMap map[string]string
+	var err error
+
+	if _, err := os.Stat(filepath.Join(projectPath)); os.IsNotExist(err) {
+		return nil, fmt.Errorf("%s. Your project path is invalid", errorStr)
+	}
+
+	if rocksVersionsMap, err = common.LuaGetRocksVersions(projectPath); err != nil {
+		return nil, fmt.Errorf("%s. %s. See --project-path flag", err, errorStr)
+	}
+
+	if len(rocksVersionsMap) == 0 {
+		return nil, fmt.Errorf(`%s. Looks like your project directory
+does not contain a .rocks directory... Did your built your project?`, errorStr)
```

Comment:
`Did your` -> `Did you`.

I sure that we should separate cases "path is not a project" and "path is a project without .rocks".

I think it's OK to think that "path is a project" if it contains a rockspec.